selenium_tests/lti_emailer/page_objects/canvas_masquerade_page_object.py

```diff
@@ -0,0 +1,35 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base_page_object import EmailerBasePageObject
+from urlparse import urljoin
+
+
+class CanvasMasqueradePageLocators(object):
+    CONFIRM_BUTTON = (By.CSS_SELECTOR, "a.btn.masquerade_button")
+    MASQUERADE_URL = 'users/{}/masquerade'
+
+
+class CanvasMasqueradePageObject(EmailerBasePageObject):
```

Comment:
Good point.  Masquerade is a service. Elliottt originally had it originally structured within a folder -- I see what he was doing there now.  I'll restore that structure (same for PIN service) - thanks!